Modules/cjkcodecs/_codecs_cn.c

```diff
@@ -279,7 +279,8 @@ DECODER(gb18030)
             REQUIRE_INBUF(4);
             c3 = INBYTE3;
             c4 = INBYTE4;
-            if (c < 0x81 || c3 < 0x81 || c4 < 0x30 || c4 > 0x39)
+            if (c < 0x81 || c > 0xFE || c3 < 0x81 || c3 > 0xFE ||
+                c4 < 0x30 || c4 > 0x39)
```

Comment:
The two different standards are compatible (ignore the draft one, it's not a standard) and doesn't make any difference here.

And about links, I think the only one worth considering here is the wikipedia one, others are all non-authoritative. The truly authoritative one is reserved by the company and you have to pay for it :-(. But if anyone is interested in it, google GB18030 will lead you to wikipedia.